### ISSUE TITLE
Create an API around Inline object

### ISSUE THREAD

**Issue by Mogztter** [asciidoctor/asciidoctor.js#272]
https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor/inline.rb

- [ ] `text` -> `getText`
- [ ] `type` -> `getType`
- [ ] `target` -> `getTarget`

**Comment by Mogztter** [comment]
Actually this is already implemented. 
@mojavelinux Do you know if it's possible to get an inline node from the AST ? I'm trying to write a test case to use methods on an Inline object.

**Comment by Mogztter** [comment]
I believe this is not possible because Asciidoctor does not parse inline syntax. To test the API we will need to create an `Inline`: 

```rb
Inline.new parent, context, text, opts
```

**Comment by mojavelinux** [comment]
> Do you know if it's possible to get an inline node from the AST ?

Yes and no. The inline node exists for a fraction of a second between when it is created by the substitutor and when the convert method is called on it. So if you get into the internals of Asciidoctor, you can snag it. But it's not accessible from the public API yet.
